weaviate/embedded.py

```diff
@@ -31,9 +31,10 @@ class EmbeddedOptions:
     persistence_data_path: str = os.environ.get("XDG_DATA_HOME", DEFAULT_PERSISTENCE_DATA_PATH)
     binary_path: str = os.environ.get("XDG_CACHE_HOME", DEFAULT_BINARY_PATH)
     version: str = "1.21.1"
-    port: int = 6666
+    port: int = 8079
```

Comment:
don't know if it makes sense to expose `DEFAULT_PORT = 8079` and `DEFAULT_GRPC_PORT = 50060` constants and use them here, wdyt?